src/codegen.cpp

```diff
@@ -357,6 +357,7 @@ static Function *jlboundserrorv_func;
 static Function *jlcheckassign_func;
 static Function *jldeclareconst_func;
 static Function *jlgetbindingorerror_func;
+static Function *jlboundp_func;
```

Comment:
not urgent, we should do a naming-cleanup-pass of the exported API before 1.0 and the unexported APIs after